libbpf-tools/trace_helpers.h

```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+#ifndef __TRACE_HELPERS_H
+#define __TRACE_HELPERS_H
+
+struct ksym {
+	long addr;
+	char name[0];
+};
+
+typedef struct ksyms ksyms_t;
```

Comment:
Let's follow kernel guidelines here and not create unnecessary typedefs. There is nothing wrong with `struct ksyms`.

Reply:
Thanks, I'll read  kernel guidelines to learn more about typedefs.